src/type.js

```diff
@@ -564,6 +527,10 @@ Sk.builtin.type.prototype.tp$setattr = function (pyName, value) {
     // class attributes are direct properties of the object
     var jsName = pyName.$jsstr();
     this[jsName] = value;
+    this.prototype[jsName] = value;
```

Comment:
Should these get the reserved-word rewrites?

Reply:
Resolved in commit 272954faea8d465080a3c2fe069671d9617f49d4